index.js

```diff
@@ -28,7 +28,17 @@ const exampleMovies = require("./movies");
       "James and the Giant Peach",
     ];
  */
-function getAllMovieTitles() {}
+function getAllMovieTitles(movies) {
+  //Return array of all movie titles (movies.title).
+  //Return [] if the inputted array is empty.
+
+  let movieTitles = [];
+
+  for(let cinema of movies) {
```

Comment:
I like where you were going with this but the convention here would be to name this `movie` as opposed to `cinema`. Nice work spotting the `for...of` loop use case here.